PayvoHQ/wallet
# refactor: update welcome banners

## Summary

New welcome banners shared by oleg

To test:
- create a new account so you see the welcome modal and see the images on the different steps,
- Try with different accent-colors

## Checklist
